packages/graBing/PKGBUILD

```diff
@@ -0,0 +1,36 @@
+# This file is part of BlackArch Linux ( http://blackarch.org ).
+# See COPYING for license details.
+
+pkgname="graBing"
+pkgver=9.bb8b08b
+pkgrel=1
+pkgdesc="Counts all the hostnames for an IP adress"
+arch=('x86_64' 'i686')
+url="https://github.com/black-brain/graBing"
+license=('GPL3')
+groups=('blackarch-fingerprint' 'blackarch-scanner')
+source=('git+https://github.com/black-brain/graBing')
+depends=('python2')
+makedepends=('git')
+provides=("graBing")
+conflicts=("graBing")
+replaces=("graBing")
+sha1sums=('SKIP')
+
+prepare() {
+	cd "$pkgname"
+
+	# Eliminating carriage return and specifying python version
+	sed -i -e "s|\r$||" -e "s|usr/bin/env python|/usr/bin/env python2|" ${pkgname}.py
+}
+
+pkgver() {
+  cd "$pkgname"
+
+  echo $(git rev-list --count HEAD).$(git rev-parse --short HEAD)
+}
+
+package() {
+	cd "$pkgname"
```

Comment:
Is it working? Shouldn't it be:
`cd "${srcdir}/${pkgname}"`

Reply:
For a reading in [Creating Packages ArchWiki](https://wiki.archlinux.org/index.php/creating_packages) and here [man PKGBUILD](http://jlk.fjfi.cvut.cz/arch/manpages/man/PKGBUILD.5) I understand that before execute that function the pwd changes to `$srcdir` isn't it?